tests/installation/enable_selinux.pm

```diff
@@ -28,13 +29,26 @@ sub run {
         send_key 'ret';
     }
 
-    send_key 'alt-m';
-    send_key_until_needlematch 'security-selinux-enforcing', 'down';
-    send_key 'ret' if $textmode;
+    if (is_microos('<5.3')) {
```

Comment:
What is this condition doing? We don't have any microOS with version 5.3. 
If this is to be applied to SLE Micro, it should be `is_sle_micro('<5.3')`